### ISSUE TITLE
AAE in data augmentation

### ISSUE THREAD

**Issue by mia-7** [eriklindernoren/PyTorch-GAN#181]
As I first touch the GAN, I wanna know how to use aae to data augmentation? Directly input an random matrix  compling with Normal(0, 5) distribution into the decoder? or revise the loss function?
Hoping for your reply! Thank you very much
